### PR TITLE
Automate Kicks and Bans

### LINKED ISSUE #568: Increase automated bans
Currently after 3 warns in a week, its a 7-day ban (**Change this to 2 weeks**)
I also want to kick the user and notify them the 4th time (no matter the time span)
And ban someone the >=5th time (no time limit)

It takes a lot to get to 5 warns, there has to be something wrong to get there

### PR DESCRIPTION
**Pull request by tanndlin**
fixes #568

### CODE REVIEW

File: src/services/moderation.service.ts [hunk 1]
```diff
@@ -285,12 +343,28 @@ export class ModService {
     return 'Banned User';
   }
 
+  private async _kickUser(user: discord.GuildMember) {
+    await user?.send(
+      'You are being kicked for too many warnings\n' +
+        `You currently have been warned ${this._KICK_THRESH} times. On the ${this._BAN_THRESH}th you will be banned permanently`
```
Comment (c650) [inline src/services/moderation.service.ts:349]: nit: 'th' is not always correct. Prefer language like "After X warnings,"

File: src/services/moderation.service.ts [hunk 2]
```diff
@@ -377,11 +451,10 @@ export class ModService {
     return await this._writeDataToFile(data);
   }
 
-  private async _getBanStatus(guild: Guild, id: string): Promise<string> {
+  private async _getBanStatus(guild: discord.Guild, id: string): Promise<string> {
```
Comment (c650) [inline src/services/moderation.service.ts:454]: why not import {Guild,...} from discord ?

**Review by c650 — APPROVED** [review]

File: src/services/moderation.service.ts (hunk 2 above)
Comment (tanndlin) [reply src/services/moderation.service.ts:454]: There are so many imports the import statement is like 8 lines

Comment (c650) [reply src/services/moderation.service.ts:454]: thats fine lol

**Pull request merged**